test/test_ops.py

```diff
@@ -388,6 +466,101 @@ def test_vmapvjp(self, device, dtype, op):
             for loop_out, batched_out in get_fallback_and_vmap_exhaustive(fn, args, {}):
                 self.assertEqual(loop_out, batched_out, atol=1e-4, rtol=1e-4)
 
+    # There are several variations we care about
+    # 1) primal batched (TODO)
+    # 2) tangent batched (batched grads) <--
+    # 3) both batched (TODO)
+    # The below tests (2) only.
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_vmapjvp', {
+        # These should to be fixed:
+
+        # Random failures: Copied from above
+        xfail('nn.functional.dropout'),  # randomness
+        xfail('double'),
+        xfail('linalg.householder_product'),
+
+        # Batching rule not implemented for aten::_local_scalar_dense
+        xfail('linalg.eigh'),
+        xfail('linalg.tensorinv'),
+        xfail('linalg.inv'),
+        xfail('linalg.matrix_power'),
+        xfail('linalg.cholesky'),
+
+        # Batching rule not implemented for aten::logical_{and,or}.out
```

Comment:
Are these problems in PyTorch core? (I would expect them to be)

Reply:
These were due to me testing batched forward in addition to batched grads... but it is fixed now